### ISSUE TITLE
Are there plans to implement drop-in replacements for parts of sklearn.feature_selection?

### ISSUE THREAD

**Issue by nirizr** [dask/dask-ml#138]
Some (like `VarianceThreshold`) may be pretty easy to do, others can be a little more complex and may require more thought.

I'm wondering if there's a plan to cover that as part of dask-ml?

**Comment by TomAugspurger** [comment]
Yes, some of those would definitely be in scope. Looking through the list, the Recursive ones probably can't be parallelized.

The various `Select*`  ones should be doable, assuming the score functions can be implemented.

Do you have any interest in working on these? It can be done in bits and pieces. I suspect that you're correct that `VarianceThreashold` is the most straightforward.

**Comment by nirizr** [comment]
I expect to be needing some soon, at least `VarianceThreshold` to get us started. I was going to implement it directly in our source code but if its in-scope I would definitely prefer putting it here.

We're also using a slightly naive DT implementation on top of Dask, let me know if you will be interested that as well and I'll open a PR and we can continue discussing the details there.

**Comment by TomAugspurger** [comment]
Great!

What's DT stand for?

**Comment by nirizr** [comment]
sorry, I meant implementing a random forest based on parallelizing decision trees using dask.
